### ISSUE TITLE
Support Languages

### ISSUE THREAD

**Issue by amosayomide05** [iamtraction/google-translate#41]
Add a list of supported languages with their short form to your readme.md to make it easier to developers to use ☺️

Like English - en

**Comment by iamtraction** [comment]
yeah i should do that. here's a link to where you can find it for now: https://github.com/iamtraction/google-translate/blob/3de9ce69d4aacd9c53624219d4463ad26a64d437/src/languages.js#L10-L116

**Comment by amosayomide05** [comment]
> yeah i should do that. here's a link to where you can find it for now:
> 
> https://github.com/iamtraction/google-translate/blob/3de9ce69d4aacd9c53624219d4463ad26a64d437/src/languages.js#L10-L116

Ok cool
